packages/esm-patient-medications-app/src/types.ts

```diff
@@ -1,27 +1,26 @@
-import { OpenmrsResource } from '@openmrs/esm-framework';
 import type { Drug, OrderBasketItem } from '@openmrs/esm-patient-common-lib';
 
 export interface DrugOrderBasketItem extends OrderBasketItem {
-  drug: Drug;
-  unit: DosingUnit;
-  commonMedicationName: string;
-  dosage: number;
-  frequency: MedicationFrequency;
-  route: MedicationRoute;
-  quantityUnits: QuantityUnit;
-  patientInstructions: string;
   asNeeded: boolean;
   asNeededCondition: string;
-  startDate: Date | string;
-  durationUnit: DurationUnit;
+  commonMedicationName: string;
+  dateActivated: Date | string;
```

Comment:
Apologies from the diff noise from reordering these properties alphabetically. The key change here is renaming this property from `startDate` to `dateActivated`.